extensions/2.0/Khronos/KHR_materials_specular/README.md

```diff
@@ -0,0 +1,177 @@
+# KHR\_materials\_specular
+
+## Khronos 3D Formats Working Group
+
+* TODO
+
+## Acknowledgments
+
+* TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds two parameters to the metallic-roughness material: `specular` and `specularColor`.
+
+`specular` allows users to configure the strength of the specular reflection in the dielectric BRDF. A value of zero disables the specular reflection, resulting in a pure diffuse material. The metal BRDF is not affected by the parameter.
+
+`specularColor` changes the F0 color of the specular reflection in the dielectric BRDF, allowing artists to use effects known from the specular-glossiness material (`KHR_materials_pbrSpecularGlossiness`) in the metallic-roughness material.
+
+
+## Extending Materials
+
+The strength of the specular reflection is defined by adding the `KHR_materials_specular` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_specular": {
+                    "specularFactor": 1.0,
+                    "specularColorFactor": [1.0, 1.0, 1.0],
+                    "specularTexture": 0,
+                }
+            }
+        }
+    ]
+}
+```
+
+Factor and texture are combined by multiplication to describe a single value.
+
+| |Type|Description|Required|
+|-|----|-----------|--------|
+| **specularFactor** | `number` | The strength of the specular reflection. | No, default: `1.0`|
+| **specularTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A grayscale texture that defines the specular factor. Will be multiplied by specularFactor. | No |
```

Comment:
I know that the current materials use a factor to scale texture values - however I would just like to raise the question if this property is used, eg how many models have an occlusion texture AND and the strength factor?

For this extension the question is simply - does anyone forsee using a specularTexture with a specularFactor other than 1.0?
Or can the specularTexture * specularFactor multiplication be removed?